server/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -536,18 +536,16 @@ public void testCopyToWithWhereNoMatch() throws Exception {
 
     @Test
     public void testCopyFromNestedArrayRow() throws Exception {
-        // assert that rows with nested arrays aren't imported
+        // assert that rows with nested arrays are imported
```

Comment:
I think we can remove the comment completely, since now the method name describes what is tested.